bank_statement_check_number/tests/test_check_number.py

```diff
@@ -0,0 +1,38 @@
+from odoo.tests import tagged
+
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+
+
+@tagged("post_install", "-at_install")
+class TestCheckNumber(AccountTestInvoicingCommon):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        cls.bank_journal = cls.company_data["default_journal_bank"]
+        cls.currency = cls.company_data["currency"]
+        cls.statement = cls.env["account.bank.statement"].create(
+            {
+                "name": "test_statement",
+                "line_ids": [
+                    (
+                        0,
+                        0,
```

Comment:
```suggestion
                    (
                        0,
                        0,
```

please apply the new syntax by using `Command.create()`